scripts/src/admin/wrapper.html

```diff
@@ -333,15 +333,25 @@ <h1><a href="/">Marketplace Admin</a></h1>
 				const appId = editor.frame.parentNode.dataset.appId;
 				try {
 					json = editor.get();
+					const isLimitsNumbers = (obj, limitName) => typeof obj.limits[limitName] === 'number';
+					if (!json.limits || !Object.keys(json.limits).every(isLimitsNumbers.bind(null, json))) {
+						toast('config limits got incorrect format')
+						return false;
+					}
+
 					const response = await fetch(`/applications/${appId}/config`, {
 						method: 'PUT',
 						body: JSON.stringify(json),
 						headers: new Headers({'Content-Type': 'application/json'})
 					})
 					if (response.ok) {
 						closeEditor(editor)
+					} else {
+						toast('config is incorrect')

```

Comment:
You do:
`config is incorrect (rejected by server)` which will help us debug this in the future.